lib/rankings_router.js

```diff
@@ -60,7 +59,7 @@ router.use('/rankings.csv', (req, res) => {
       .concat(Array.from(new Array(maxScoresCount), (val, index) => index + 1))
     const csv = [headers]
       .concat(ranks.map(rank => [ rank.rank, rank.team.number, rank.highest ]
-        .concat(rank.scores.map(score => score === NO_SHOW_KEY ? NO_SHOW_SCORE : score))))
+        .concat(rank.scores.map(score => score === NO_SHOW_KEY ? NO_SHOW_KEY : score))))
```

Comment:
This substitution was specifically requested. Whenever you change a piece of coffee, ask yourself why it was done that way in the beginning. I'd you don't know, ask the person who wrote it. Please resume using this line, of you want you can change the name to `CSV_NO_SHOW_SCORE`.